test/Helper/_files/modules/application/views/scripts/partialLoopChildObject.phtml

```diff
@@ -1,12 +1,12 @@
 <?php
 
-$vars = $this->vars();
+$vars = (array) $this->vars();
 
 if (empty($vars)) {
 	echo "No object model passed";
 } elseif (isset($vars['message'])) {
 	echo $vars['message'];
 } else {
-	$objKey = current($this->vars())->helper->getObjectKey();
+	$objKey = current($vars)->helper->getObjectKey();
```

Comment:
Nice,

This BC break on ArrayAccess object in 7.4 is terrible
Already bitten on another project, not easy to catch